tests/capi/doc/chfl_property/vector3d.c

```diff
@@ -0,0 +1,20 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) Guillaume Fraux and contributors -- BSD license
+
+#include <chemfiles.h>
+#include <assert.h>
+
+int main() {
+    // [example]
+    CHFL_PROPERTY* property = chfl_property_vector3d((chfl_vector3d){0, 3.2, -1});
+
+    chfl_vector3d value = {0};
```

Comment:
Should be
`chfl_vector3d value = {0., 0., 0.};`
right?

Reply:
Missing value in the brace are replaced by 0 automatically: see http://en.cppreference.com/w/c/language/array_initialization, " Initialization from brace-enclosed lists" section. This is really neat when working with a lot of arrays =)